src/stock_server.c

```
@@ -1,34 +1,256 @@
+static stock_status_t conn_queue_init(conn_queue_t *queue);
+static void conn_queue_destroy(conn_queue_t *queue);
+static void conn_queue_insert(conn_queue_t *queue, int connfd);
+static int conn_queue_remove(conn_queue_t *queue);
+static stock_status_t create_workers(pthread_t *tids, conn_queue_t *queue,
+                                     int *created);
+static void join_workers(pthread_t *tids, int count);
+static void stop_workers(conn_queue_t *queue, pthread_t *tids, int count);
+static void *worker_main(void *arg);
+static void handle_client(int connfd);
+
+/*
```

Cppcheck (2.20.0)

[information] Include file

(missingIncludeSystem)